Library/Formula/deheader.rb

```diff
@@ -1,12 +1,16 @@
 class Deheader < Formula
   homepage "http://www.catb.org/~esr/deheader"
-  url "http://www.catb.org/~esr/deheader/deheader-1.0.tar.gz"
-  sha1 "dc9fc816af1631a84ace7f94a85a3a424d72dbed"
-  head "https://git.gitorious.org/deheader/deheader.git"
+  url "http://www.catb.org/~esr/deheader/deheader-1.1.tar.gz"
+  sha256 "69f69e9c7d9398221cb49f7de91df0d122e4b0ec942bede2d7c592401e4b913c"
+  head "git://thyrsus.com/repositories/deheader.git"
+
+  depends_on :python
```

Comment:
Roughly: Applications and well-behaved language bindings never receive `depends_on :python`. We often assume that `python` is Python 2.7; then, use `depends_on :python if MacOS.version <= :snow_leopard` (:python is only satisfied if `python` is 2.7; system python is 2.7 on all OS versions newer than Snow Leopard). Anything that installs with an explicit linkage against a Python framework (i.e. `otool -L some_module.(dylib|so)` mentions `Python` or `libpython`) needs a) to be fixed to link with `-undefined dynamic_lookup` instead of `-framework Python` and b) to depend_on :python in the meantime.